### PR TITLE
Updating

### PR DESCRIPTION
**Pull request by ivanMise**

### CODE REVIEW

**Comment by ivanMise** [comment]
RF07-TC-002 CREATE EQUIPMENT

image


image

**Comment by ivanMise** [comment]
image

File: src/main/java/org/umssdiplo/automationv01/core/managepage/RegisterEquipment/RegisterEquipment.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    private WebElement equipmentDesc;
+
+    @FindBy(id="imageEquipment")
+    private WebElement equipmentImage;
+
+    @FindBy(id="newEquipment")
+    private WebElement createBtn;
+
+    public void registerEquipment() {
+        String equiName = PropertyAccessor.getInstance().getEquiName();
```
Comment (marcelogaray) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/RegisterEquipment/RegisterEquipment.java:39]: slipt your steps in this method

File: src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/StepsDefinitionSSID.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    public void setMyCredentialsOnLoginPage() throws Throwable {
+        login.setCredentials();
+    }
+
+    @And("^register equipment on 'Equipaments/create' page$")
+    public void registerEquipment() throws Throwable {
+        System.out.println("==========================================");
+        System.out.println("Testing SSID RF07-TC-002 CREATE EQUIPMENT");
+        System.out.println("==========================================");
+        this.registerEquipment = new RegisterEquipment();
```
Comment (marcelogaray) [inline src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/StepsDefinitionSSID.java:28]: in the "step definition" you dont need to create the Page Object.
for this pull request just you need to send "registerEquipment()" method of the RegisterEquipment class.

File: src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/StepsDefinitionSSID.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    private RegisterEquipment registerEquipment;
+
+    @Given("^'SSID' page is loaded$")
+    public void ssidPageIsLoaded() throws Throwable {
+        login = LoadPage.loginPage();
+    }
+
+    @And("^set my credentials on SSID 'Login' page$")
+    public void setMyCredentialsOnLoginPage() throws Throwable {
+        login.setCredentials();
```
Comment (marcelogaray) [inline src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/StepsDefinitionSSID.java:20]: once you have the credentials you need some page object Dashboard/Header/MenuMain/etc etc etc

**Review by marcelogaray — CHANGES_REQUESTED** [review]
review observations

**Pull request closed without merging**